### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adisky**
**What this PR does / why we need it**:
This PR updates the readme file, about all the plugins we host here.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
The doc releated to openstack ccm is moved to doc/openstack-cloud-controller-manager.md, it has not been updated, it will be updated in a seperate PR.

**Release note**:

```release-note
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,27 @@
...
-- <https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180530-cloud-controller-manager.md>
-- <https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager>
-- <https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/>
-
-## Examples
-
-Here are some examples of how you could leverage `openstack-cloud-controller-manager`:
-
-- [loadbalancers](examples/loadbalancers/)
+* [OpenStack Cloud Controller Manager](examples/loadbalancers/)
```
Comment (jichenjc) [inline README.md]: a typo...

Comment (adisky) [reply README.md]: ohh!! Thanks will update

File: docs/openstack-cloud-controller-manager.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- <https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180530-cloud-controller-manager.md>
+- <https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager>
+- <https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/>
+
+## Examples
+
+Here are some examples of how you could leverage `openstack-cloud-controller-manager`:
+
+- [loadbalancers](examples/loadbalancers/)
```
Comment (ramineni) [inline docs/openstack-cloud-controller-manager.md]: link gives 404 Not found ..

File: docs/openstack-cloud-controller-manager.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Developing
```
Comment (ramineni) [inline docs/openstack-cloud-controller-manager.md]: This should ideally go into getting started guide. wdyt?

File: docs/openstack-cloud-controller-manager.md (hunk 2 above)
Comment (adisky) [reply docs/openstack-cloud-controller-manager.md]: thanks, for pointing it will update

File: docs/openstack-cloud-controller-manager.md (hunk 3 above)
Comment (adisky) [reply docs/openstack-cloud-controller-manager.md]: yes, we need to refactor this doc also, i just updated Readme, this will taken care in separate PR

**Comment by adisky** [comment]
@ramineni @jichenjc Thanks for review will update shortly!!

**Comment by adisky** [comment]
@liu-sheng why retest there was no need??

**Comment by ramineni** [comment]
@jichenjc comments yet to be addressed ? 
/hold

**Comment by jichenjc** [comment]
/lgtm cancel
sorry, I might comment on the wrong PR

**Comment by adisky** [comment]
@ramineni @jichenjc  PR is updated PTAL!!!

File: docs/openstack-cloud-controller-manager.md (hunk 2 above)
Comment (adisky) [reply docs/openstack-cloud-controller-manager.md]: Done

File: README.md [hunk 4]
```diff
@@ -1,57 +1,27 @@
...
-- <https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180530-cloud-controller-manager.md>
-- <https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager>
-- <https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/>
-
-## Examples
-
-Here are some examples of how you could leverage `openstack-cloud-controller-manager`:
-
-- [loadbalancers](examples/loadbalancers/)
+* [OpenStack Cloud Controller Manager](/docs/openstack-cloud-controller-manager.md/)
```
Comment (ramineni) [inline README.md]: this link would be accessible once code gets merged i suppose , currently it is giving 404 . Right?

Comment (adisky) [reply README.md]: @ramineni i also checked this, for a moment it is showing the page and then 404, so i think, yes it will be accessible after code merge

Comment (ramineni) [reply README.md]: ok

**Comment by ramineni** [comment]
Thanks for the update.
/lgtm

**Comment by adisky** [comment]
cc @dims @adisky @hogepodge

**Comment by adisky** [comment]
cc @dims @hogepodge @lingxiankong

**Comment by jichenjc** [comment]
thanks for the update, LGTM as well

**Pull request merged**